src/tools/bpf_link_helper/xdp_onload_prepare.c

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL-2.0
+
+/* xdp_onload_prepare -- load an XDP ELF object with xsk map and program
+ *
+ * cd $SOMEDIR
+ * git clone --recurse-submodules https://github.com/libbpf/bpftool.git
+ * cd bpftool
+ * export BPFTOOL=$PWD
+ * make
```

Comment:
```suggestion
 * make -C libbpf/src
```
There's no Makefile in the top directory, hence "diving" into `libbpf/src`. I haven't yet verified, but `make -C src` could be a better one! The version I've checked out is https://github.com/libbpf/bpftool/commit/22621e39da611c292b7c13d89af7bdccc7b905fb.

Reply:
Thanks @ivatet-amd .

I must have messed up the comments. These changes look good to me.

Come to think of it, if the only thing needed from bpftool is libbpf, then it may be cleaner to just add that dependency (https://github.com/libbpf/libbpf).

+1 on the suggested changes. If we want to change more thoroughly, I can do that too.

Reply:
Thanks @wdebruij. It seems the `libbpf` alone is sufficient. No worries, I'll apply it too. 👍